app/models/booking.rb

```diff
@@ -1,6 +1,6 @@
 class Booking < ApplicationRecord
   validates :start_date, :end_date, presence: true
-  validates :end_date_after_start_date
+  # validates :end_date_after_start_date
```

Comment:
Why is this validation commented out ?

Reply:
When I tried to run seed file without that commented, it came back with an error that said "Bookings require at least one validation" - after commenting that line it worked all fine, so i imagine the syntax may be wrong when calling validates on a custom method. Instead of looking into resolving this one I thought best to get the seeds done and move on as we can look at this custom validation later on or remove it entirely if we wish to do so.

Reply:
Fair there probably isn't much value in stressing over these validations now - so we can delete it but I'll make a card on trello to capture it and it can be a task for later down the track